<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSE</class>
 <widget class="QDialog" name="frmSE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto Unidad 3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

font-family: Microsoft JhengHei;
font-size: 14px;
background-color: white
}

QPushButton
{

background-color: rgb(23,161,165);
border-radius: 10px;
color: white;

}

QPushButton:hover
{
background-color: rgb(77,183,187);
border-radius: 10px;
color: white;
}

QPushButton:pressed
{
background-color: rgb(23,161,165);
border-radius: 10px;
color: white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnAceptar">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Preprocesamiento:</string>
   </property>
   <widget class="QRadioButton" name="rB_media">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Media</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_moda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Moda</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_mediana">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mediana</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_vMayor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Mayor</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_vMenor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Menor</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Preprocesamiento:</string>
   </property>
   <widget class="QRadioButton" name="rB_normalizacion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalización</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_estandarizacion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estandarización</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_complemento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Complemento</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Preprocesamiento-Outliers</string>
   </property>
   <widget class="QRadioButton" name="rB_IQR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IQR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_PuntZ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Puntuaciones Z</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string>ARDUINO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>PYTHON</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>PYTHON</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
